### DISCUSSION TITLE
【求助】有推荐的希人毛发护理品吗？

### DISCUSSION THREAD

**Discussion by A-chllenge** [share121/inter-knot#3479] [其他]
![1](https://github.com/user-attachments/assets/3e9a3ecb-ada0-47ea-a1b8-266e9805f55a)
身为希人而且还是兽型希人的我，每天要大力毛发真的很麻烦，尤其是春秋交接和快入冬的时候都会因为掉毛而发生过敏的情况<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by 00zzxx** [comment]
吃点鱼油吧，我家小猫就吃，虽然没什么用，但是心理上会轻松一些。<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
